infrastructure/service-bus.tf

```diff
@@ -19,6 +19,13 @@ resource "azurerm_servicebus_subscription_rule" "topic_filter_rule_sscs" {
   }
 }
 
+resource "azurerm_servicebus_subscription_rule" "hmctsDeploymentId" {
+  name            = "hmc_to_sscs_subscription_rule"
+  subscription_id = module.servicebus-subscription.id
+  filter_type     = "SqlFilter"
+  sql_filter      = "${var.deploymentId} ='' OR ${var.deploymentId} = hmctsDeploymentId"
```

Comment:
is this filter combined with the filter above when picking up messages?

Reply:
Yeah they should both be applied as two different filters on the subscription